modules/grpc/bigquery/bigquery-dest.cpp

```diff
@@ -184,6 +234,68 @@ DestinationDriver::construct_schema_prototype()
   this->schema_prototype = msg_factory.GetPrototype(this->schema_descriptor);
 }
 
+bool
+DestinationDriver::load_protobuf_schema()
+{
+  this->protobuf_schema.src_tree = std::make_unique<google::protobuf::compiler::DiskSourceTree>();
+  this->protobuf_schema.src_tree->MapPath(this->protobuf_schema.proto_path, this->protobuf_schema.proto_path);
+
+  this->protobuf_schema.error_coll = std::make_unique<ErrorCollector>();
+
+  this->protobuf_schema.importer =
+    std::make_unique<google::protobuf::compiler::Importer>(this->protobuf_schema.src_tree.get(),
+                                                           this->protobuf_schema.error_coll.get());
+
+  const google::protobuf::FileDescriptor *file_descriptor =
+    this->protobuf_schema.importer->Import(this->protobuf_schema.proto_path);
+
+  if (!file_descriptor || file_descriptor->message_type_count() == 0)
+    {
+      msg_error("Error initializing BigQuery destination, protobuf-schema() file can't be loaded",
+                log_pipe_location_tag(&this->super->super.super.super.super));
+      return false;
+    }
+
+  this->schema_descriptor = file_descriptor->message_type(0);
```

Comment:
[optional]

We could support choosing a `Message` by name, which can be useful if there are multiple in the proto file (but that might not even be possible in the case of BigQuery, I don't know).

Reply:
Cool!

Consider this and the include related comment resolved.